screenpipe-audio/src/vad_engine.rs

```diff
@@ -225,6 +249,34 @@ impl VadEngine for SileroVad {
         Ok(status == VadStatus::Speech && result.prob > threshold)
     }
 
+    fn audio_type(&mut self, audio_chunk: &[f32]) -> anyhow::Result<VadStatus> {
```

Comment:
@EzraEllette fyi we might want to remove webrtc altogether, which does not make sense to use anyway and is causing issues on some OS

https://github.com/mediar-ai/screenpipe/issues/442